pycode/memilio-surrogatemodel/memilio/surrogatemodel/ode_secir_groups/network_architectures.py

```diff
@@ -20,87 +20,203 @@
 import tensorflow as tf
 
 
-def mlp_multi_input_single_output(num_age_groups=6):
-    """ Simple MLP Network which takes the compartments for multiple time steps as input and
-    returns the 8 compartments for all six age groups for one single time step.
+def mlp_multi_input_single_output(num_age_groups=6, num_outputs=8, num_hidden_layers=3, num_neurons_per_layer=32, activation='relu'):
```

Comment:
will skip this for now and review this when we adress this in the next pull request